aries-site/src/examples/templates/feedback/PowerOnExample.js

```diff
@@ -0,0 +1,205 @@
+import React, { useCallback, useContext, useEffect, useState } from 'react';
+import {
+  Anchor,
+  Box,
+  Button,
+  Card,
+  Layer,
+  Form,
+  Footer,
+  FormField,
+  Heading,
+  Spinner,
+  StarRating,
+  ResponsiveContext,
+  Text,
+  TextArea,
+} from 'grommet';
+import { StatusGoodSmall } from 'grommet-icons';
+
+const FeedbackLayout = ({ children, onClick, onClose }) => {
+  const breakpoint = useContext(ResponsiveContext);
+  return (
+    // feedback layout
+    <Layer>
```

Comment:
We should try to reuse: https://github.com/grommet/hpe-design-system/tree/master/aries-site/src/examples/templates/ModalDialog